sdk/ai/azure-ai-projects/samples/agents/sample_agents_json_schema.py

```diff
@@ -0,0 +1,115 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+
+"""
+DESCRIPTION:
+    This sample demonstrates how to use agents with JSON schema output format.
+
+USAGE:
+    python sample_agents_json_schema_async.py
+
+    Before running the sample:
+
+    pip install azure-ai-projects azure-identity pydantic
+
+    Set this environment variables with your own values:
+    PROJECT_CONNECTION_STRING - the Azure AI Project connection string, as found in your AI Studio Project.
+"""
+
+import os
+
+from enum import StrEnum
+from pydantic import BaseModel, TypeAdapter
+from azure.ai.projects import AIProjectClient
+from azure.identity import DefaultAzureCredential
+from azure.ai.projects.models import (
+    MessageTextContent,
+    MessageRole,
+    ResponseFormatJsonSchema,
+    ResponseFormatJsonSchemaType,
+    RunStatus,
+)
+
+# Create an Azure AI Client from a connection string, copied from your AI Studio project.
+# At the moment, it should be in the format "<HostName>;<AzureSubscriptionId>;<ResourceGroup>;<HubName>"
```

Comment:
Should <HubName> be <ProjectName>?

Reply:
Removed this code as it was removed from all other samples.